components/dropdown/src/auro-dropdown.js

```diff
@@ -361,12 +363,14 @@ export class AuroDropdown extends LitElement {
         <div class="slotContent">
           <slot @slotchange="${this.handleDefaultSlot}"></slot>
         </div>
+        <div id="bibSizer" part="size"></div>
         <auro-dropdownbib
           id="bib"
           role="tooltip"
           ?common="${this.common}"
           ?rounded="${this.common || this.rounded}"
-          ?inset="${this.common || this.inset}">
+          ?inset="${this.common || this.inset}"
+          mobileFullscreenBreakpoint="${this.mobileFullscreenBreakpoint}">
```

Comment:
```suggestion
          .mobileFullscreenBreakpoint="${this.mobileFullscreenBreakpoint}">
```

Adding the `.` should make it so the attribute is only added if there is a value for it.

Reply:
To clarify,... using `.` is for string attributes and `?` is for boolean attributes. They affectively do the same thing in that they only add the attribute when there is a not a false/null/undefined value to apply.